core/domain/topic_services.py

```diff
@@ -132,6 +132,22 @@ def get_all_topic_summaries():
     return topic_summaries
 
 
+def get_all_triaged_skills():
+    """Returns the ids of all the skills that are linked to some topics.
+
+    Returns:
+        list(str). The ids of all the skills linked to some topic.
+    """
+    skill_ids = []
+    all_topic_models = topic_models.TopicModel.get_all()
+    for topic_model in all_topic_models:
```

Comment:
We should convert to domain objects before doing anything with topics. Never operate on models directly.

Also maybe implement topic.get_all_categorized_skill_ids() or something like that.

Also, consider maintaining skill_ids as a set in the first place to avoid the extra set() cast at the end. (Though you might want to do an analysis of the time complexity of both approaches first.) I expect we'll have on the order of 10k skills in the long run, so we should be a bit careful here.

Reply:
Done.